docs/platform/tutorials/currency-alerting.md

```diff
@@ -1,256 +1,242 @@
 
-!!! note
+Let’s set up each function one by one, starting with our input function.
 
-	CoinAPI is a platform that provides data APIs for many financial instruments including cryptocurrency. We will be using it to get live Bitcoin (BTC) data.
 
-Let’s create a free coinAPI account:
+### Setting Up The CoinAPI Input Function
 
-1.  Go to [https://www.coinapi.io/](https://www.coinapi.io/){target=_blank}.
+**Purpose:**  _Ingest BTC/USD Exchange Rate Data_
 
-2.  Click the "Get a free API key" button and complete the dialog to
-    create a free account.
+This function connects reads a live stream of updates for the currency pair: `BTC/USD`. We get this data by connecting to the [CoinAPI](https://www.coinapi.io/) [Websocket](https://en.wikipedia.org/wiki/WebSocket) (the free [sandbox version](https://docs.coinapi.io/#endpoints-2)). 
 
-3.  You will receive your API key on your provided email. **Keep it safe
-    for later**.
+* The function streams the exchange rate data to a topic called `currency-exchange-rates`. 
+* Downstream functions can then read fom this topic and process it in different ways. 
+* In this case, we're going to check the price against a threshold.
 
-### Create a free Twilio account
 
-!!! note
+**To set up the CoinAPI input function, follow these steps:**
 
-	Twilio is a developer platform for communications. We can use Twilio, for instance, to send SMS messages.
+1. In the [Quix Portal](https://portal.platform.quix.ai/), open the Quix Library<br>![Library icon](../images/tutorials/currency-alerting/library-icon.png)
+   <br><br>
+2. In the search box on the library page, search for "_CoinAPI—Exchange Rate Feed_".<br>
+   You should see a result that resembles the following example: ![CoinAPI library item](../images/tutorials/currency-alerting/coinapi.png "CoinAPI library item")
+   <br><br>
 
-Let’s create a free Twilio account:
+3. Click **Preview Code**, and on the page that appears, click **Edit Code**.
+    * Optionally, you can also jump straight to **Setup & Deploy** but we want to take a quick look at what the code is doing first, and perhaps change a few things.
+    * When you choose to edit a library item, Quix prompts you to create a copy of it as a Project (this is because library items are read-only).
+   <br><br>
 
-1.  Go to [https://www.twilio.com/](https://www.twilio.com/){target=_blank}.
+4. On the 'Project Creation' page, set the following fields and environment variables:
 
-2.  Click the "Sign Up" button and complete the dialog. Do the email and
-    text message verifications.
+    | Field      | Value                          |
+    | ----------- | ------------------------------------ |
+    | **Name:**      | Enter a project name or keep the default suggestion. |
+    | **Output:**      | Select the output topic—in this case, select `currency exchange rates` from the list|
```

Comment:
> Select the output topic—in this case, select `currency exchange rates` from the list|

End text with period and add a space before the vertical bar (pipe symbol). 

> topic—in

Could we surround the dash with spaces, or perhaps just have this as two sentences? For example:

Select the output topic. In this case, select...